src/components/TodoForm.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Form, Button } from 'react-bootstrap';
+
+interface ITodoFormProps {
+  total: number;
+  addTodo: Function;
+}
```

Comment:
這邊明確的定義傳進來的 function 要長什麼樣子
void 代表沒有回傳值

```suggestion
interface ITodoFormProps {
  total: number;
  addTodo: (id: number, title: string) => void;
}
```